source/blogs/dbc/dbc.rst

```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
-   changes to take effec.
+    .. code-block:: bash
 
-   .. image:: images/trigger_udev.png
+        root@intel:~# udevadm control --reload-rules
+        root@intel:~# udevadm trigger
 
+#. Rename or remove the kernel module *usb_debug.ko* in the host system if any.
 
-4. Rename or remove the usb_debug.ko driver in the Host System if available.
+    .. code-block:: bash
 
-   .. image:: images/rename_driver.png
+        root@intel:~# cd /lib/modules/$(uname -r)/kernel/drivers/usb/serial/ && mv usb_debug.ko usbdebug
 
+#. Check whether the *usb_debug* driver module is loaded to the kernel with the following command:
 
-5. Unload usb_debug driver module if it's already loaded.
+    .. code-block:: bash
 
-   .. image:: images/unload_driver.png
+        root@intel:~# lsmod | grep usb_debug
 
+#. Unload the *usb_debug* driver module if it's already loaded.
 
-6. This you can check using below command.
+    .. code-block:: bash
 
-   .. image:: images/list_driver.png
+        root@intel:~# rmmod usb_debug
 
+#. The ``adb`` command installed by the Android SDK does not support ADB over DbC, you should use the ``adb`` command built from the |C| source tree. The ADB over DbC enabled ``adb`` command is avaiable in the *out/host/linux-x86/bin/* folder after the build.
 
-7. Default installed adb binary present in the Host System will not work for ADB
-   over DbC, you will need to use adb binary available under below build directory.
+Steps to enable ADB over DbC support on |NUC| BLKNUC7iDNHE system
+-----------------------------------------------------------------
 
-   .. image:: images/adb_host_binary.png
+#. Check the Android property value *persist.vendor.sys.usb.adbover* with the following command. The default value is ``dwc``, represents normal ADB over USB (DWC).
 
-Steps to enable ADB over DbC support in Cedalon KBL-NUC (Product Code: BLKNUC7iDNHE)
-------------------------------------------------------------------------------------
+    .. code-block:: bash
 
-1. Check the below android property value, default this have value dwc, which means normal 
-   ADB over USB(DWC) will work.
+        root@intel:~# getprop persist.vendor.sys.usb.adbover
+        dwc
 
-   .. image:: images/getprop_dwc.png
+#. Reset the property value to ``dbc``, then reboot the target system.
 
-2. If step 1 output is "dwc" then set it to "dbc".
+    .. code-block:: bash
 
-   .. image:: images/setprop_dbc.png
+        root@intel:~# setprop persist.vendor.sys.usb.adbover dbc
 
-3. Now restart the target system.
+Connect the Target to the Host System
+-------------------------------------
 
-Connection between Host and Target Machine
-------------------------------------------
+Plug the debug Target to the Host system using a `USB Type-A to Type-A (3.0) SuperSpeed Debug cable <https://www.datapro.net/products/usb-3-0-super-speed-a-a-debugging-cable.html>`_. 
+USB 2.0 Type-A to Type-A cable doesn't work in this case.
```

Comment:
Suggested rewrite: 
A USB 2.0 Type-A to Type-A cable does not work in this case.